configs/LLNL-Magma-Penguin-icelake-OmniPath/variables.yaml

```diff
@@ -6,7 +6,7 @@
 variables:
   timeout: "120"
   scheduler: "slurm"
-  sys_cores_per_node: "96"
+  sys_cores_per_node: "36"
```

Comment:
Since Magma really does have 96 cores, it's probably best not to change this value and, instead, create a new config for the 36-core system you'll be using.

Reply:
@gracenansamba please remove the changes to the magma config from this PR.  We should be using the Dane config instead (which I just added - please test).